src/openms/source/IONMOBILITY/IMDataConverter.cpp

```diff
@@ -197,6 +197,7 @@ namespace OpenMS
         result.addSpectrum(std::move(in[k]));
       }
     }
+    in.clear(); // TODO: @cbielow. What about exp meta values?
```

Comment:
@cbielow better to clear MSExperiment instead of having it in a mixed state? What about meta values? these are currently not transferred? is this correct

Reply:
ExperimentalSettings are currently not copied. This should probably be fixed.
Metadata of spectra are already copied in `splitByIonMobility`.

Reply:
do we need to update ranges?

Reply:
not at the moment since all data relevant to ranges is shuffled around but not modified.
However, once we include `IonMobility` in the range, we'd technically need to update, because there could be binning of IM values...